### ISSUE TITLE
[MAS1.4.11] [Visual Requirements-Live Chat Dialog] Luminosity Ratio is less than 3:1 for border outline with respective background color for “Send button and File upload button".

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1962]
**Actual Result:**
Luminosity Ratio is less than 3:1 for border outline with respective background color for “Send button and File upload button.

**Expected Result:**
Luminosity Ratio should be greater than or equal to 3:1 for border outline with respective background color for “Send button and File upload button.

**Note:**
This issue exits for throughout application for all button controls.
Same issue also with the "Restart Conversation Dropdown" exist on the same screen.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to "Open bot" and select it.
4. Open bot dialog gets opened and connect the pen bot 
5. Live chat screen gets opened 
6. Navigate to the “Send button and File upload button".
7. Open 'Color Contrast Analyser' application.
8. Observe the issue

**User Impact:**
Users with low vision or color blindness needs that difference between contrast of text is always high.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=6d787213-d391-4513-b643-14ad134a9aec](url)

**Attachment:**
![CCA failed for Send button and File upload button](https://user-images.githubusercontent.com/55279608/68101596-6b77cf00-fef4-11e9-8d3b-a87c60608de5.png)

**Comment by tonyanziano** [comment]
@corinagum @compulim This will have to be fixed in Web Chat.

The restart button chevron color has been fixed in the Emulator. The color for the chevron is now `#4A4A4A` as seen in the code here:

[light.css](https://github.com/microsoft/BotFramework-Emulator/blob/master/packages/app/client/src/ui/styles/themes/light.css#L228)
[neutral.css](https://github.com/microsoft/BotFramework-Emulator/blob/master/packages/app/client/src/ui/styles/themes/neutral.css#L10)

![image](https://user-images.githubusercontent.com/3452012/70384004-a70b1c00-192c-11ea-8ab9-cc5d3080268f.png)

**Comment by corinagum** [comment]
Is this using an older version of the Emulator? The Web Chat colors are not up to date. The icons in the latest code are `#767676`, and pass color contrast requirements:
![image](https://user-images.githubusercontent.com/14900841/70459347-d8890200-1a68-11ea-86fa-7271cbdbbe15.png)

**Comment by tonyanziano** [comment]
Let me double check, and see if we are providing some incorrect style options or something on the Emulator side.

When I checked the colors were not meeting the 4.5:1 requirement.

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
